CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+## [Unreleased]
+
+## [0.5.0] - 2022-09-05
+
+- Add caching support (optionally) to avoid hitting the IDP every request.
+
+## [0.4.0] - 2022-09-02
+
+- Add unauthentication endpoint to remove access/refresh tokens and
```

Comment:
the word `unauthentication`. doesnt exists, you can use `unauthenticated`

Reply:
Fixed, at the end I used the word `logout`.